tests/app/phase-team-response/issues-pending/issues-pending.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { IssuesPendingComponent } from '../../../../src/app/phase-team-response/issues-pending/issues-pending.component';
+import { Issue } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+import { IssueService } from '../../../../src/app/core/services/issue.service';
+import { UserService } from '../../../../src/app/core/services/user.service';
+import { USER_Q } from '../../../constants/data.constants';
+
+fdescribe('IssuesPendingComponent', () => {
+    describe('.ngOnInit', () => {
+        const dummyTeam: Team = new Team({
+            id: 'dummyId',
+            teamMembers: [],
+          });
+        const dummyIssue: Issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+        const issueService: IssueService = new IssueService(null, null, null, null, null, null);
+        issueService.updateLocalStore(dummyIssue);
+        const userService: UserService = new UserService(null, null);
+        userService.currentUser = USER_Q;
+        const issuesPendingComponent: IssuesPendingComponent = new IssuesPendingComponent(issueService, null, userService);
+        const DUMMY_DUPLICATE_ISSUE_ID = 1;
+        const DUMMY_RESPONSE = 'dummy response';
+
+        beforeEach(() => {
+            issuesPendingComponent.ngOnInit();
+        });
+
+        it('should set default filter to return false for duplicate issues with no team response', () => {
```

Comment:
```suggestion
        it('should set filter to return false for a duplicate issue with no team response', () => {
```
Within the context of the IssuesPendingComponent, let's simply refer to the function as `filter` (since that's the property name within the component).

The "default filter" terminology comes from [IssuesDataTable](https://github.com/CATcher-org/CATcher/blob/0a37660eadcc77a07296a8e19c8ca88e9c83ea19/src/app/shared/issue-tables/IssuesDataTable.ts#L20). The filter function defined in `IssuesPendingComponent` gets set as the `defaultFilter` of the IssuesDataTable instance that serves as the data source for the "pending issues" table in team response phase.

Regarding the change from `issues` to `issue`: the filter function technically operates on a single issue.

Reply:
Yup, will change!